radioactive-decay/exercises/q_radioactive_decay.yml

```diff
@@ -0,0 +1,5 @@
+uid: q-radioactive decay 
+title: Radioactive Decay
+type: Derivation
+options:
+  text: Derive the Poisson distribution from the binomial distribution.
```

Comment:
(meta-comment: This may be a place where reviews can get kind of contentious.... )  Does this assessment actually test the learning objective?  This assessment is a purely mathematical/statistical operation.